src/ui/components/UploadScreen/Privacy.tsx

```diff
@@ -46,10 +46,10 @@ function FavIcon({ url }: { url: string }) {
   return (
     <div className="relative">
       <div className="flex">
-        <MaterialIcon>public</MaterialIcon>
+        <MaterialIcon className="invisible">public</MaterialIcon>
       </div>
       <img
-        className="h-4 w-4 absolute top-0 left-0 bg-white"
+        className="h-4 w-4 absolute top-1 left-1 bg-white"
```

Comment:
What's the reason for adding the adding top/left values? It looks like it ends up uncentering the images in the row.

![image](https://user-images.githubusercontent.com/15959269/151870425-afbd238e-5136-4582-bb53-895e8d10895e.png)

Reply:
Yeah that looks bad. I was seeing the opposite problem, maybe it was a browser difference or something? To address.